components/Document/DocumentOptions.tsx

```diff
@@ -49,14 +55,48 @@ function downloadPDF(pdfUrl) {
 }
 
 const DocumentOptions = ({ document: doc, metadata }: Props) => {
+  const alert = useAlert();
   const currentUser = useSelector((state: RootState) =>
     isEmpty(state.auth?.user) ? null : parseUser(state.auth.user)
   );
   const documentContext = useContext(DocumentContext);
   const { revalidateDocument } = useCacheControl();
 
   const isModerator = Boolean(currentUser?.moderator);
-  const isHubEditor = Boolean(currentUser?.author_profile?.is_hub_editor);
+  const isHubEditor = Boolean(currentUser?.authorProfile?.isHubEditor);
+
+
+  const handleRemoveContent = () => {
+    alert.show({
+      // @ts-ignore
+      text: (
+        <div>
+          {`Permanently delete this ${doc.unifiedDocument.documentType}? This cannot be undone.`}
+        </div>
+      ),
+      buttonText: "Yes",
+      onClick: async () => {
+        censorDocument({
+          unifiedDocumentId: doc.unifiedDocument.id,
+          onSuccess: () => {
+            alert.show({
+              // @ts-ignore
+              text: (
+                <div>
+                  {`This ${doc.unifiedDocument.documentType} has been deleted.`}
+                </div>
+              ),
+              buttonText: "OK",
+              onClick: () => {
+                window.location.reload();
+              },
```

Comment:
Shouldn't it redirect somewhere else (e.g. home)? I tried to flow in staging and it didn't seem like the question/post successfully deleted since it just refreshed the page and I still saw the question/post.

I'm also able to still edit the question after it was supposedly "deleted." Perhaps need to clarify what actually happened to the user?

Reply:
Agree this should use something like nextjs router. Ill check it out